packages/pcd-types/src/pcd.ts

```diff
@@ -143,6 +127,32 @@ export interface PCDPackage<
   deserialize(seralized: string): Promise<PCD<C, P>>;
 }
 
+/**
+ * When displaying a {@link PCD} in Zupass, the PCDUI methods will be used to generate a
+ * card and, optionally, a header (used only when {@link PCDPackage.getDisplayOptions} does
+ * not return a header).
+ *
+ * @typeParam {@link C} the type of {@link PCD.claim} for the {@link PCD} encapsulated
+ *   by this {@link PCDUI}
+ *
+ * @typeParam {@link P} the type of {@link PCD.proof} for the {@link PCD} encapsulated
+ *   by this {@link PCDUI}
+ */
+export interface PCDUI<C = any, P = any> {
```

Comment:
A thought that came to mind is that ostensibly there could be different pcds that have the same claim and proof types, and this type setup has no way to disambiguate between the two. Not an actionable thought I think for now.

Reply:
Another thing that comes to mind but is not necessarily actionable is that the default types for `C` and `P` should probably be `unknown` rather than `any`. That may entail a larger refactor though.

Reply:
This is just copied from `PCDPackage`, minus the types for args and init. I'll be honest and admit that I've never understood what the generic aspect of `PCDPackage` (or `SerializedPCD` etc.) is for. I can imagine having a `PCD` reference where I don't know the "subclass", and I can imagine having a concrete PCD reference, but I can't imagine how I would ever want a reference to a `PCD<something, something, something, something>`.